packages/solidity-contracts/deploy/beta5devnet/008.fuel_message_portal_v3.ts

```diff
@@ -19,7 +21,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     new FuelMessagePortalV3(deployer),
     {
       unsafeAllow: ['constructor'],
-      constructorArgs: [MaxUint256],
+      constructorArgs: [MaxUint256, RATE_LIMIT_DURATION],
```

Comment:
Do not change deploy scripts outside of `deploy/hardhat`.

These scripts were already run in its day, they belong to older versions of the portal

Reply:
I see thanks